### PR TITLE
Add conf modules for httpd, httpd24-httpd and jbcs-httpd24-httpd

### PR DESCRIPTION
**Pull request by huali027**

### CODE REVIEW

File: uploader.json [hunk 1]
```diff
@@ -1500,6 +1505,11 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_scl_jbcs_httpd24"
         },
+        {
+            "file": "/opt/rh/jbcs-httpd24/root/etc/httpd/conf.modules.d/()*.+\\.conf",
+            "pattern": [],
+            "symbolic_name": "httpd_modules_conf_scl_jbcs_httpd24"
```
Comment (bfahr) [inline uploader.json]: @huali027 the symbolic name needs to match the spec name that you used in insights-core here, and on line 1496, it should not be a new name.

File: uploader.v2.json [hunk 2]
```diff
@@ -1500,6 +1505,11 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_scl_jbcs_httpd24"
         },
+        {
+            "file": "/opt/rh/jbcs-httpd24/root/etc/httpd/conf.modules.d/()*.+\\.conf",
+            "pattern": [],
+            "symbolic_name": "httpd_modules_conf_scl_jbcs_httpd24"
+        },
```
Comment (bfahr) [inline uploader.v2.json]: Same comment as above.

File: uploader.json (hunk 1 above)
Comment (huali027) [reply uploader.json]: @bfahr, sure, I see, I'll update it, thanks.

**Comment by huali027** [comment]
@bfahr, I update this pr to uploader_json branch directly, please help to review again when you have time, thanks.

File: uploader.json [hunk 3]
```diff
@@ -1480,6 +1480,11 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_d"
```
Comment (bfahr) [inline uploader.json]: This is part of the spec that you are changing and also needs to be the same spec name, `httpd_conf`.  Would you mind to please change this spec too.

File: uploader.v2.json [hunk 4]
```diff
@@ -1480,6 +1480,11 @@
             "pattern": [],
             "symbolic_name": "httpd_conf_d"
```
Comment (bfahr) [inline uploader.v2.json]: @huali027 would you mind to change this spec name to match the actual spec name in your core change, `httpd_conf`. Thanks!

Comment (huali027) [reply uploader.v2.json]: > @huali027 would you mind to change this spec name to match the actual spec name in your core change, `httpd_conf`. Thanks!

@bfahr, yes, sure, I'll udpate it in this patch too.

**Pull request merged**